src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -196,8 +203,10 @@ def generate_MCP_1_default_config(self, filename):
         config = {
             'iterationData': [iter1_config, iter2_config],
             'sasHarnessesConfigs': [sas_harness_0, sas_harness_1],
```

Comment:
`sasTestHarnessConfigs`

Reply:
Modified,   'sasHarnessesConfigs' to 'sasTestHarnessesConfigs'